src/main/java/org/ltr4l/cli/Predict.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2018 org.LTR4L
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ltr4l.cli;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.cli.*;
+import org.ltr4l.Ranker;
+import org.ltr4l.Version;
+import org.ltr4l.evaluation.RankEval;
+import org.ltr4l.nn.MLP;
+import org.ltr4l.nn.RankNetMLP;
+import org.ltr4l.query.Document;
+import org.ltr4l.query.Query;
+import org.ltr4l.query.QuerySet;
+import org.ltr4l.tools.Config;
+import org.ltr4l.tools.Report;
+import org.ltr4l.trainers.MLPTrainer;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.lang.invoke.MethodHandles;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class Predict {
```

Comment:
Can you write PredictTest class that tests Predict? You can use TrainTest as a reference.